### DISCUSSION TITLE
Mobile Automation - Android

### DISCUSSION THREAD

**Discussion by Nikhil-Waghmare** [getgauge/gauge#2190] [General]
Hi All, Can we use Gauge to do automation of native apps or can we integrate appium+.js + Gauge just like how we have java + Gauge + Appium combination.

**Comment by sriv** [comment]
you should be able to use appium+js+gauge, in theory, as long as appium has javascript support. I don't have an example however. Perhaps @saikrishna321 has some experience?

**Reply by saikrishna321** [reply]
Official Js client is WDIO for Appium. You can use WebdriverIO and gauge together through a remote client and not as a service.

**Reply by saikrishna321** [reply]
Sample WDIO with web tests - https://github.com/pradeep0601/gauge-wdio-x-browser-test

**Reply by Nikhil-Waghmare** [reply]
> Official Js client is WDIO for Appium. You can use WebdriverIO and gauge together through a remote client and not as a service.

I am a beginner, how exactly to use it with a remote client. Any documentation.
